.github/workflows/cypress.yml

```diff
@@ -40,15 +40,10 @@ jobs:
         with:
           node-version: ${{ steps.get-node-version.outputs.NODE_VERSION }}
 
-      - name: Cache dependencies
-        id: cache-dependencies
-        uses: actions/cache@v2
+      - name: Install Chrome
+        uses: browser-actions/setup-chrome@latest
         with:
-          path: |
-            ~/.cache/yarn
-            ~/.cache/Cypress
-            node_modules
-          key: ${{ steps.get-node-version.outputs.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}
+          chrome-version: stable
```

Comment:
Is installing it necessary? I think the self-hosted runner already [has Chrome installed](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#browsers-and-drivers) since it mirrors `ubuntu-latest`.

Reply:
It already has Chrome installed, but not the latest version, which is why I've added it here.